tests/testsuite/build.rs

```diff
@@ -2266,138 +2382,12 @@ fn lto_build() {
         .file("src/main.rs", "fn main() {}")
         .build();
     p.cargo("build -v --release")
-        .with_stderr(
-            "\
-[COMPILING] test v0.0.0 ([CWD])
-[RUNNING] `rustc --crate-name test --edition=2015 src/main.rs [..]--crate-type bin \
-        --emit=[..]link \
-        -C opt-level=3 \
-        -C lto \
-        [..]
-[FINISHED] `release` profile [optimized] target(s) in [..]
-",
-        )
-        .run();
-}
-
-#[cargo_test]
-fn verbose_build() {
```

Comment:
I don't understand these `verbose_xxx_build` test cases. They doesn't seem to check anything important except verbose outputs, which are already covered a lot by other tests.